### DISCUSSION TITLE
Installation not working

### DISCUSSION THREAD

**Discussion by StefanBu1970** [tijsverkoyen/HomeAssistant-FusionSolar#168] [General]
I am trying to integarte the package to HomeAssitant, but the option to install the packes is missing and the packeage is not found by the integrations search. (or devices, etc)
User is Admin
Any ideas???
Regards,
Stefan

**Comment by tijsverkoyen** [comment]
Have you installed HACS?
Once you gave installed HACS you can follow the steps in https://github.com/tijsverkoyen/HomeAssistant-FusionSolar?tab=readme-ov-file#installation

**Comment by StefanBu1970** [comment]
Hi Tijs,

 

I had some trouble installing HACS. Now the integration is working perfect, thanks for Your quick reply.

 

Kind regards

 

Stefan

 

Von: Tijs Verkoyen ***@***.***> 
Gesendet: Donnerstag, 2. Mai 2024 21:49
An: tijsverkoyen/HomeAssistant-FusionSolar ***@***.***>
Cc: Stefan Burek ***@***.***>; Author ***@***.***>
Betreff: Re: [tijsverkoyen/HomeAssistant-FusionSolar] Installation not working (Discussion #168)

 

Closed #168 <https://github.com/tijsverkoyen/HomeAssistant-FusionSolar/discussions/168>  as resolved.
